<html>
<head>
  <title>local_channel</title>
  <style type="text/css">
    a, a:active {
      text-decoration: none; color: blue;
    }
    a:visited {
      color: #48468F;
    }
    a:hover, a:focus {
      text-decoration: underline; color: red;
    }
    body {
      background-color: #F5F5F5;
    }
    h2 {
      margin-bottom: 12px;
    }
    th, td {
      font: 100% monospace; text-align: left;
    }
    th {
      font-weight: bold; padding-right: 14px; padding-bottom: 3px;
    }
    th.tight {
        padding-right: 6px;
    }
    td {
      padding-right: 14px;
    }
    td.tight {
        padding-right: 8px;
    }
    td.s, th.s {
      text-align: right;
    }
    td.summary {
      white-space: nowrap;
      overflow: hidden;
    }
    td.packagename {
      white-space: nowrap;
      text-overflow: ellipsis;
      overflow: hidden;
      max-width: 180px;
      padding-right: 8px;
    }
    td.version {
      //white-space: nowrap;
      overflow: hidden;
      max-width: 90px;
      padding-right: 8px;
    }
    table {
      background-color: white;
      border-top: 1px solid #646464;
      border-bottom: 1px solid #646464;
      padding-top: 10px;
      padding-bottom: 14px;
    }
    address {
      color: #787878;
      padding-top: 10px;
    }
  </style>
</head>
<body>
  <h2>local_channel</h2>
  <h3><a href="rss.xml">RSS Feed</a>&nbsp;&nbsp;&nbsp;<a href="channeldata.json">channeldata.json</a></h3>
<a href="linux-64">linux-64</a>&nbsp;&nbsp;&nbsp;<a href="noarch">noarch</a>&nbsp;&nbsp;&nbsp;  <table>
    <tr>
      <th style="padding-right:18px;">Package</th>
      <th>Latest Version</th>
      <th>Doc</th>
      <th>Dev</th>
      <th>License</th>
<th class="tight">linux-64</th><th class="tight">noarch</th>      <th>Summary</th>
    </tr>
    <tr>
      <td class="packagename"><a href="https://github.com/LTLA/celldex" alt="bioconductor-celldex">bioconductor-celldex</a></td>
      <td class="version">1.4.1</td>
      <td></td>
      <td></td>
      <td class="tight">GPL-3</td>
<td>X</td><td></td>      <td class="summary">Provides a collection of reference expression datasets with curated...</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://bioconductor.org/packages/devel/bioc/html/GeomxTools.html" alt="bioconductor-geomxtools">bioconductor-geomxtools</a></td>
      <td class="version">3.1.1</td>
      <td></td>
      <td></td>
      <td class="tight">MIT</td>
<td>X</td><td></td>      <td class="summary">NanoString GeoMx Tools</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://bioconductor.org/packages/3.15/bioc/html/NanoStringNCTools.html" alt="bioconductor-nanostringnctools">bioconductor-nanostringnctools</a></td>
      <td class="version">1.4.0</td>
      <td></td>
      <td></td>
      <td class="tight">MIT</td>
<td>X</td><td></td>      <td class="summary">NanoString nCounter Tools</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://bioconductor.org/packages/3.14/bioc/html/SpatialDecon.html" alt="bioconductor-spatialdecon">bioconductor-spatialdecon</a></td>
      <td class="version">1.4.3</td>
      <td></td>
      <td></td>
      <td class="tight">GPL-3</td>
<td>X</td><td></td>      <td class="summary">Deconvolution of mixed cells from spatial and/or bulk gene expression data</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/NIDAP-Community/DSPWorkflow" alt="r-dspworkflow">r-dspworkflow</a></td>
      <td class="version">1.3.1</td>
      <td></td>
      <td></td>
      <td class="tight">MIT</td>
<td>X</td><td></td>      <td class="summary">A set of functions for analyzing RNA data from the spatial...</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/NIDAP-Community/DSPWorkflow" alt="r-dspworkflow-dev">r-dspworkflow-dev</a></td>
      <td class="version">1.3.1</td>
      <td></td>
      <td></td>
      <td class="tight">MIT</td>
<td>X</td><td></td>      <td class="summary">A set of functions for analyzing RNA data from the spatial...</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/NIDAP-Community/DSPWorkflow" alt="r-dspworkflow-dev-test">r-dspworkflow-dev-test</a></td>
      <td class="version">0.1.0</td>
      <td></td>
      <td></td>
      <td class="tight">MIT</td>
<td>X</td><td></td>      <td class="summary">A set of functions for analyzing RNA data from the spatial...</td>
    </tr>    <tr>
      <td class="packagename"><a href="software.broadinstitute.org/harmony" alt="r-harmony">r-harmony</a></td>
      <td class="version">1.1.0</td>
      <td></td>
      <td></td>
      <td class="tight">GPL-3</td>
<td>X</td><td></td>      <td class="summary">Implementation of the Harmony algorithm for single cell integration,...</td>
    </tr>    <tr>
      <td class="packagename"><a href="software.broadinstitute.org/harmony" alt="r-harmony-dev">r-harmony-dev</a></td>
      <td class="version">1.1.0</td>
      <td></td>
      <td></td>
      <td class="tight">GPL-3</td>
<td>X</td><td></td>      <td class="summary">Implementation of the Harmony algorithm for single cell integration,...</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/CCBR/l2p" alt="r-l2p">r-l2p</a></td>
      <td class="version">0.0_13</td>
      <td></td>
      <td></td>
      <td class="tight">Public-Domain</td>
<td>X</td><td></td>      <td class="summary">Find Pathways for HUGO gene symbols</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/CCBR/l2p" alt="r-l2psupp">r-l2psupp</a></td>
      <td class="version">0.0_13</td>
      <td></td>
      <td></td>
      <td class="tight">Public-Domain</td>
<td>X</td><td></td>      <td class="summary">Find Pathways for HUGO gene symbols</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/NIDAP-Community/nidapFunctions" alt="r-nidapfunctions">r-nidapfunctions</a></td>
      <td class="version">0.7.8</td>
      <td></td>
      <td></td>
      <td class="tight">MIT</td>
<td>X</td><td></td>      <td class="summary">Useful Tools for Developing in NIDAP Code Workbook.</td>
    </tr>    <tr>
      <td class="packagename">r-scworkflow</td>
      <td class="version">1.0.2</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td><td></td>      <td class="summary">None</td>
    </tr>    <tr>
      <td class="packagename">r-scworkflow-dev</td>
      <td class="version">1.0.2</td>
      <td></td>
      <td></td>
      <td class="tight"></td>
<td>X</td><td></td>      <td class="summary">None</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/FNLCR-DMAP/SCSAWorkflow" alt="spac">spac</a></td>
      <td class="version">0.3.3</td>
      <td><a href="https://fnlcr-dmap.github.io/SCSAWorkflow/">doc</a></td>
      <td><a href="https://github.com/FNLCR-DMAP/SCSAWorkflow">dev</a></td>
      <td class="tight">BSD</td>
<td>X</td><td></td>      <td class="summary">SPatial Analysis for single-Cell analysis (SPAC)is a Scalable Python...</td>
    </tr>    <tr>
      <td class="packagename"><a href="https://github.com/FNLCR-DMAP/SCSAWorkflow" alt="spac-dev">spac-dev</a></td>
      <td class="version">0.3.4</td>
      <td><a href="https://fnlcr-dmap.github.io/SCSAWorkflow/">doc</a></td>
      <td><a href="https://github.com/FNLCR-DMAP/SCSAWorkflow">dev</a></td>
      <td class="tight">BSD</td>
<td>X</td><td></td>      <td class="summary">SPatial Analysis for single-Cell analysis (SPAC)is a Scalable Python...</td>
    </tr>  </table>
  <address>Updated: 2024-02-05 21:35:53 +0000 - Files: 16</address>
</body>
</html>